pages/apis/rest_api/pipeline_templates.md

```diff
@@ -0,0 +1,293 @@
+# Pipeline templates API
+
+> 📘 Enterprise feature
+> [Pipeline templates](https://buildkite.com/docs/pipelines/templates) are only available on an [Enterprise](https://buildkite.com/pricing) plan.
+
+The pipeline templates API allows admins to create and manage pipeline templates for an organization.
+Non-admins can only read or assign pipeline templates marked as `available` by organization admins.
+
+## Pipeline template data model
+
+<table class="responsive-table">
+<tbody>
+  <tr><th><code>uuid</code></th><td>UUID of the pipeline template</td></tr>
+  <tr><th><code>graphql_id</code></th><td><a href="/docs/apis/graphql-api#graphql-ids">GraphQL ID of the pipeline template</a></td></tr>
+  <tr><th><code>name</code></th><td>Name of the pipeline template</td></tr>
+  <tr><th><code>description</code></th><td>Description of the pipeline template</td></tr>
+  <tr><th><code>configuration</code></th><td>YAML step configuration for the pipeline template</td></tr>
+  <tr><th><code>available</code></th><td>When set to <code>true</code>, the pipeline template can assigned to pipelines by non-admins<br><em>Default:</em> <code>false</code></td></tr>
```

Comment:
One last instance:

```suggestion
  <tr><th><code>available</code></th><td>When set to <code>true</code>, non-admins can assign the pipeline template to pipelines<br><em>Default:</em> <code>false</code></td></tr>
```

Reply:
Did a full text search on the code just to be sure.. 😆
That ought to be it!